README.md

```diff
@@ -61,6 +63,10 @@ Optional:
 
 * [gawk] for dasht-server(1) search engine
 
+[Dash.app]: https://kapeli.com/dash
+
+* [Dash.app] to let the Dash MacOS app manage your docsets
```

Comment:
There is already a named link reference to that same Dash URL in this file.  Should re-use it.